src/containers/MyProfile/MyProfile.tsx

```diff
@@ -62,9 +62,19 @@ export const MyProfile = ({ navigation }: MyProfileStackProps<"myProfile/profile
   };
 
   const handleLogout = () => {
-    handleAuthLogout?.();
-    navigation.navigate("main", {
-      screen: "main/products",
+    navigation.navigate("myProfile", {
+      screen: "logout",
+      params: {
+        onLogoutPress: async () => {
```

Comment:
Screen still isn't scrollable and dev tools has a warning 
<img width="535" alt="image" src="https://user-images.githubusercontent.com/36453308/216572424-945a7bf6-9c8f-4693-874b-bc6c0e987c57.png">
this can appears due to passing function  to the params